test/fixtures/wpt/streams/transferable/transfer-with-messageport.window.js

```diff
@@ -105,7 +105,7 @@ async function transferMessagePortWith(constructor) {
   await transferMessagePortWithOrder3(new constructor());
 }
 
-async function advancedTransferMessagePortWith(constructor) {
+async function advancedTransferMesagePortWith(constructor) {
```

Comment:
all of the changes in this file (`Message` (old) => `Mesage` (new)) seem to be incorrect? I am dyslexic so maybe I'm reading these words too hard, but autocorrect while writing this comment seems to agree.

Reply:
Reverted!